translations/LICENSE-fr.md

```diff
@@ -31,3 +31,5 @@ doit en être modifié.
      d'autre et râler quand il fonctionne mal fait de vous un connard DE PREMIÈRE.
      Tentez de trouver la solution par vous-même. Un non-connard soumettrait même le
      correctif.
+     
+  4. Si vous utilisez du code, en faire le vôtre ce serait être ROYALLEMENT un connard. [Comment citer un dépôt](https://academia-stackexchange-com.translate.goog/questions/14010/how-do-you-cite-a-github-repository?_x_tr_sl=en&_x_tr_tl=fr&_x_tr_hl=fr&_x_tr_pto=wapp). Alternativement, même un simple commentaire donnant l'attribution à l'endroit où vous avez trouvé le code serait acceptable.
```

Comment:
"En faire le votre" is a little confusing, could mean that you can't change the licensed thing.

Would something like "vous l'attribuer"

Otherwise LGTM

Reply:
+1 for "vous l'attribuer"

Reply:
Error : only one L in ROYALEMENT

Reply:
I agree "vous l'attribuer" is better.
I tried to avoid the use of "adverbe" Royallement (and stick to an  idiomatic "adjective" like the other rules) with this alternative translation: 

Si vous utilisez du code, vous l'attribuer fait de vous un connard SUPRÊME. [Comment citer un dépôt](https://academia-stackexchange-com.translate.goog/questions/14010/how-do-you-cite-a-github-repository?_x_tr_sl=en&_x_tr_tl=fr&_x_tr_hl=fr&_x_tr_pto=wapp). Alternativement, même un simple commentaire donnant l'attribution et l'endroit où vous avez trouvé le code est acceptable.